lib/mpl_toolkits/tests/test_mplot3d.py

```diff
@@ -370,26 +370,37 @@ def test_plotsurface_1d_raises():
         ax.plot_surface(X, Y, z)
 
 
-def _test_proj_make_M():
+def _test_proj_make_M(proj='persp'):
     # eye point
     E = np.array([1000, -1000, 2000])
     R = np.array([100, 100, 100])
     V = np.array([0, 0, 1])
     viewM = proj3d.view_transformation(E, R, V)
-    perspM = proj3d.persp_transformation(100, -100)
```

Comment:
it is entirely possible this test was originally backwards. It came from some commented out code at the bottom of proj3d.py. Seemed like it might have been some debugging code for the original developer. We recently lifted it out of there and put it in as a unit test because it helped us solve a problem.